provider/internal/test/suite.go

```diff
@@ -90,9 +90,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 
 		s.valkyrie.Start()
 
-		providerURLName := strings.ToLower(s.ProviderConfig.Name)
-		providerURLName = strings.ReplaceAll(providerURLName, " ", "")
-		s.ValkyrieURL = fmt.Sprintf(baseURL+"/providers/%s", providerPort, providerURLName)
+		s.ValkyrieURL = fmt.Sprintf(baseURL+"%s%s", providerPort, valkyrieConfig.ProviderBasePath, s.ProviderConfig.BasePath)
```

Comment:
Silly detail, maybe baseURL should be an argument as well since sprintf is used

Reply:
Just realized it's because `baseURL` contains formatting as well, the port is part of it